lib/queries/community.ts

```diff
@@ -44,6 +44,29 @@ export const communityInfo = (communityId: string) =>
     },
   });
 
+export const communityAdministrators = (
+  communityId: string,
+  getToken: GetToken,
+) =>
+  queryOptions({
+    queryKey: ["communityAdmins", communityId],
+    queryFn: async () => {
+      return withSpan(
+        `query:backend:community:${communityId}:administrators`,
+        async () => {
+          const token = await getToken();
+          if (!token) throw new Error("invalid clerk token");
+          const res = await zenaoClient.getCommunityAdministrators(
+            { communityId },
+            { headers: { Authorization: `Bearer ${token}` } },
+          );
+          return res.administrators;
+        },
+      );
+    },
+    initialData: [],
+  });
+
```

Comment:
this already exists in same file isn't ?

export const communityAdministratorsQuery = (
  getToken: GetToken,
  communityId: string,
)